backend/src/routes/events.ts

```diff
@@ -253,7 +253,7 @@ async function listKubernetesObjects(options: IWatchOptions) {
 }
 
 async function watchKubernetesObjects(options: IWatchOptions, resourceVersion: string) {
-    const serviceAcccountToken = getServiceAcccountToken()
+    const serviceAcccountToken = getServiceAccountToken()
```

Comment:
The const var name still has the extra "c" `const serviceAcccountToken`

Reply:
Fixed